### DISCUSSION TITLE
Maandag 09-12-2024

### DISCUSSION THREAD

**Discussion by Annevd** [rutgerkock/buurtcampus-oost#16] [Stand-ups]
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Annevd** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Ik heb vanochtend een pull request gemaakt voor het verbeterde form component. Daarna ben ik verder gegaan aan de Creative coding spike 2.
### Wat wil je deze week bereiken (aan werken)?
Ik ga verder met de creative coding opdracht en hopelijk af en toe het projectplan waar kan.
### Waar loop je tegen aan?
Nogmaals, tijd maken voor het projectplan tijdens het maken van de creative coding opdrachten.

### **Kennis delen:**

Ik zei dat ik nog iets zou delen met jullie dat ik vorige week tijdens de eerste creative coding spike had ondervonden, namelijk: `overflow: clip;`. Tijdens het bouwen van de scroll driven animations kwam ik er achter dat de `view()` methode **niet** werkte op de sections in de homepage. Dit kwam omdat er `overflow: hidden;` op de sections stond. 
In de CSS weekly newsletter had ik `overflow: clip;` voorbij zien komen, dus was dit een goed moment om het hier uit te proberen. En inderdaad, `overflow: clip` loste dit probleem op, de scroll animation werkte nu wel. Dus als jullie dit probleem ooit nog hebben, is dit een oplossing! :)

Ook om er voor te zorgen dat Harry niet voor overflow zorgt op de html x-as, zouden jullie `overflow-x: clip` of `overflow: hidden` kunnen gebruiken op de HTML @Lmikkers @rutgerkock 

[Bron](https://ishadeed.com/article/overflow-clip/?utm_source=CSS-Weekly&utm_medium=newsletter&utm_campaign=issue-599-november-20-2024&_bhlid=71db572afb9149abeee0ca0ae97ae111815fc307)

**Comment by Nazneen05x** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?

I am okey. Ik ben vorige week bezig geweest met het bouwen van de Over pagina. Ik ben klaar met bouwen, ik moet nu nog alleen testen en daarna een pull request voor feedback op mijn code.

### Wat wil je deze week bereiken (aan werken)?

Deze week ga ik verder met het menu. Ik wil deze week proberen om de bugs op te lossen. 

### Waar loop je tegen aan?

Op dit moment alleen bugs. Verder gaan de dagen echt snel!

**Comment by wingsvn** [comment]
**Hoe gaat het met je en waar ben je nu mee bezig?**
Het gaat niet zo super, zit een beetje in een dip.

**Wat wil je deze week bereiken?**
Ehm, omdat ik laag in mijn energie zit, doe ik wat me lukt.

**Comment by Lmikkers** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Ik was vandaag niet op school door persoonlijke omstandigheden, heb het gevoel dat ik achterloop en maar niet bij kom. Moet meer focussen op school en bieb in bloei. Ik was van het weekend druk bezig met stage opdracht, had hier een beetje stress over. Deze heb ik wel ingeleverd en mijn herkansing afgelopen vrijdag, dus kan nu weer vol focussen op school!

### Wat wil je deze week bereiken (aan werken)?
Ik ga verder met de deeltaak, alleen moet nog aan deel 2 beginnen wat vandaag werd uitgelegd. Wil weer op schema komen met schoolopdracht en bieb in bloei niet vergeten en ook een paar taken oppakken en dit noteren in het projectboard/ issues.

### Waar loop je tegen aan?
Ik loop achter en moet weer bijkomen
